Dockerfile

```diff
@@ -12,23 +13,35 @@ ENV PYTHONDONTWRITEBYTECODE 1
 ENV PYTHONUNBUFFERED 1
 ENV DJANGO_ALLOWED_HOSTS *
 
-# Install psycopg2 and bpy dependencies
-RUN apt-get update \
-    && apt-get install -y --no-install-recommends build-essential postgresql gcc \
-    && apt-get remove -y build-essential \
-    && apt-get install -y --no-install-recommends linux-headers-generic g++ nodejs npm \
+RUN pip install --upgrade pip virtualenv
+RUN virtualenv /venv
+
+COPY requirements.txt .
+RUN /venv/bin/pip install -r requirements.txt
+
+COPY . $ROBOPROP_CLIENT
+RUN apt-get update && apt-get install nodejs npm -y
+RUN npm install && npx tailwindcss -i $ROBOPROP_CLIENT/static/src/input.css -o $ROBOPROP_CLIENT/static/src/output.css
+RUN rm -rf node_modules
```

Comment:
nitpick but a space between the RUN and the new image would be nice for readability

Reply:
Thank you, done